mudz.Common/Domain/GameEngine/GameRequest.cs

```diff
@@ -8,7 +9,7 @@ public class GameRequest
     {
         public GameActions GameAction { get; set; }
         public RoomKey RoomKey { get; set; }
-        public IGameObject Sender { get; set; }
+        public IPlayer Sender { get; set; }
```

Comment:
I haven't _played_ the whole game, but I couldn't see anywhere that this wouldn't be an `IPlayer`?